dotnet/src/SemanticKernel.Core/Functions/SKFunction.cs

```diff
@@ -11,13 +12,56 @@
 namespace Microsoft.SemanticKernel;
 #pragma warning restore IDE0130
 
-#pragma warning disable format
-
 /// <summary>
-/// Static helpers to create <seealso cref="ISKFunction"/> instances.
+/// Factory methods for creating <seealso cref="ISKFunction"/> instances.
 /// </summary>
 public static class SKFunction
 {
+    /// <summary>
+    /// Creates an <see cref="ISKFunction"/> instance for a method, specified via a delegate.
+    /// </summary>
+    /// <param name="method">The method to be represented via the created <see cref="ISKFunction"/>.</param>
+    /// <param name="pluginName">The optional name of the plug-in associated with this method.</param>
+    /// <param name="functionName">Optional function name. If null, it will default to one derived from the method represented by <paramref name="method"/>.</param>
+    /// <param name="description">Optional description of the method. If null, it will default to one derived from the method represented by <paramref name="method"/>, if possible (e.g. via a <see cref="DescriptionAttribute"/> on the method).</param>
+    /// <param name="parameters">Optional parameter descriptions. If null, it will default to one derived from the method represented by <paramref name="method"/>.</param>
+    /// <param name="returnParameter">Optional return parameter description. If null, it will default to one derived from the method represented by <paramref name="method"/>.</param>
+    /// <param name="loggerFactory">The <see cref="ILoggerFactory"/> to use for logging. If null, no logging will be performed.</param>
+    /// <returns>The created <see cref="ISKFunction"/> wrapper for <paramref name="method"/>.</returns>
+    public static ISKFunction Create(
+        Delegate method,
```

Comment:
nit: should the parameter name be delegate or function to be aligned with the `functionName` parameter?

Reply:
The "method" here is about the .NET method being wrapped.
The "functionName" parameter refers to the name to use for ISKFunction.